utils/jupp/Makefile

```diff
@@ -0,0 +1,143 @@
+$(call Package/jupp/Default)
+  TITLE:=WordStar-inspired text editor
+  DEPENDS:=+PACKAGE_libncurses:libncurses
+endef
+
+define Package/jupp/description
+jupp is a WordStar-inspired text and hex editor,
+for programmers and regular end users.
+endef
+
+define Package/jupp/chmod_plus_x
+	# work around automagic check for +x
+	chmod +x ${PKG_BUILD_DIR}/configure
+endef
+Hooks/Configure/Pre += Package/jupp/chmod_plus_x
+
+CONFIGURE_ARGS+= \
+	--disable-selinux \
+	--disable-termidx \
+	--enable-sysconfjoesubdir=/jupp
+
+ifeq (,${CONFIG_PACKAGE_libncurses})
+CONFIGURE_ARGS+= \
+	--disable-curses \
+	--disable-termcap \
+	--disable-terminfo
+endif
+
+define Package/jupp/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/joe $(1)/usr/bin/jupp
+endef
+
+define Package/jupp-data
+$(call Package/jupp/Default)
+  TITLE:=jupp Emacs/JOE/Pico emulations, syntax highlighting
+  PKGARCH:=all
+  DEPENDS:=jupp
+  CONFLICTS:=joe
+endef
+
+define Package/jupp-data/install
+	$(INSTALL_DIR) $(1)/etc/jupp
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/jmacsrc $(1)/etc/jupp/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/joerc $(1)/etc/jupp/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/jpicorc $(1)/etc/jupp/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/jstarrc $(1)/etc/jupp/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/jupprc $(1)/etc/jupp/
+	$(INSTALL_DIR) $(1)/etc/jupp/charmaps
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/charmaps/klingon $(1)/etc/jupp/charmaps/
+	$(INSTALL_DIR) $(1)/etc/jupp/syntax
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/asm.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/c.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/conf.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/csh.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/diff.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/fortran.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/html.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/java.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/lisp.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/mail.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/mason.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/pascal.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/perl.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/php.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/python.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/sh.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/tcl.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/tex.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/verilog.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/vhdl.jsf $(1)/etc/jupp/syntax/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/syntax/xml.jsf $(1)/etc/jupp/syntax/
```

Comment:
can this just be *.jsf? I see he list of files in the source is ~abotu this long, are you explicitly leaving something out?

Reply:
Indeed, it can be all JSF files. I’ll adjust accordingly.